tests/jobs/test_cli.py

```diff
@@ -313,7 +331,8 @@ def test_get_job_21(jobs_api_mock):
         runner = CliRunner()
         runner.invoke(cli.get_cli, ['--job-id', '1', '--version', '2.1'])
         assert jobs_api_mock.get_job.call_args == mock.call('1', version='2.1')
-        assert echo_mock.call_args[0][0] == pretty_format(LIST_21_RETURN['jobs'][0])
+        assert echo_mock.call_args[0][0] == pretty_format(
+            LIST_21_RETURN['jobs'][0])
```

Comment:
Can be undone? Or did you see linting problems?

Reply:
I just ran the linter and I think it updated this, I'll undo it